src/common/src/config.rs

```diff
@@ -284,8 +284,8 @@ mod default {
         8
     }
 
-    pub fn unsafe_worker_node_parallel_degree() -> usize {
-        4
+    pub fn worker_node_parallelism() -> usize {
+        num_cpus::get()
```

Comment:
Would this function in std lib have the same functionality as num_cpus? https://doc.rust-lang.org/stable/std/thread/fn.available_parallelism.html